easybuild/easyconfigs/s/STAR-Fusion/STAR-Fusion-1.6.0-GCC-8.2.0-2.31.1-Perl-5.28.1-Python-3.7.2.eb

```diff
@@ -31,248 +31,60 @@ dependencies = [
-        'checksums': ['1c70e438cec1f7f99a5cccd4529efb4ee0fb7ca958ca885ebf09952015b957aa'],
-    }),
-    ('Module::CoreList', '5.20190620', {
-        'source_tmpl': 'Module-CoreList-%(version)s.tar.gz',
-        'source_urls': ['https://cpan.metacpan.org/authors/id/B/BI/BINGOS/'],
-        'checksums': ['319092da2ca6432549dd054da43a349ec6afd29a1ac3e8be779c130d283b2159'],
-    }),
-    ('Module::Load', '0.34', {
-        'source_tmpl': 'Module-Load-%(version)s.tar.gz',
-        'source_urls': ['https://cpan.metacpan.org/authors/id/B/BI/BINGOS/'],
-        'checksums': ['7f5b0c4d3cc8ba867639bff61ad857a940970ea246dd3b0a6729dd7fc84059f3'],
-    }),
-    ('Module::Load::Conditional', '0.68', {
-        'source_tmpl': 'Module-Load-Conditional-%(version)s.tar.gz',
-        'source_urls': ['https://cpan.metacpan.org/authors/id/B/BI/BINGOS/'],
-        'checksums': ['7627b55cd94a2f1a4667f9203e2c990cad015bf34ec6b41b4f73af848f0698fd'],
-    }),
-    ('IPC::Cmd', '1.02', {
-        'source_tmpl': 'IPC-Cmd-%(version)s.tar.gz',
-        'source_urls': ['https://cpan.metacpan.org/authors/id/B/BI/BINGOS/'],
-        'checksums': ['502129252aabce96e5442143c4363dc7440b6dcf1a79a7fd245908b62f067063'],
-    }),
-    ('ExtUtils::MakeMaker', '7.36', {
-        'source_tmpl': 'ExtUtils-MakeMaker-%(version)s.tar.gz',
-        'source_urls': ['https://cpan.metacpan.org/authors/id/B/BI/BINGOS/'],
-        'checksums': ['06cef6429315cdc6afa9b2dc6fbdfa33538b6f68c827f441294621858e28c558'],
-    }),
-    ('ExtUtils::CBuilder', '0.280231', {
-        'source_tmpl': 'ExtUtils-CBuilder-%(version)s.tar.gz',
-        'source_urls': ['https://cpan.metacpan.org/authors/id/A/AM/AMBS/'],
-        'checksums': ['e6e72546ca0468c10bb50d4aba2a9b3908e9fe4d7708e7ee800327be075eeeeb'],
-    }),
-    ('ExtUtils::ParseXS', '3.35', {
-        'source_tmpl': 'ExtUtils-ParseXS-%(version)s.tar.gz',
-        'source_urls': ['https://cpan.metacpan.org/authors/id/S/SM/SMUELLER/'],
-        'checksums': ['41def0511278a2a8ba9afa25ccab45b0453f75e7fd774e8644b5f9a57cc4ee1c'],
-    }),
-    ('ExtUtils::CppGuess', '0.19', {
-        'source_tmpl': 'ExtUtils-CppGuess-%(version)s.tar.gz',
-        'source_urls': ['https://cpan.metacpan.org/authors/id/E/ET/ETJ/'],
-        'checksums': ['4cc631bab662848ea7c8899e0c62cdfdb367f2fe205b5bd85dfc1082672a6ad3'],
-    }),
-    ('AutoLoader', '5.74', {
-        'source_tmpl': '%(name)s-%(version)s.tar.gz',
-        'source_urls': ['https://cpan.metacpan.org/authors/id/S/SM/SMUELLER/'],
-        'checksums': ['2fac75b05309f71a6871804cd25e1a3ba0a28f43f294fb54528077558da3aff4'],
-    }),
-    ('Set::IntervalTree', '0.12', {
-        'source_tmpl': 'Set-IntervalTree-%(version)s.tar.gz',
-        'source_urls': ['https://cpan.metacpan.org/authors/id/S/SL/SLOYD/'],
-        'checksums': ['6fd4000e4022968e2ce5b83c07b189219ef1925ecb72977b52a6f7d76adbc349'],
+        'checksums': ['924f8e2b4e3cb3d8b26246b5f9c07cdaa4b8800cef345fa0811d72930d73a54e'],
```

Comment:
@lexming any idea what changed here?
better to keep both the old and new checksum in any case.

Reply:
Nothing has changed, this checksum is just moving from line 49.

Reply:
ah ok, sorry for the noise.